test/instrumentation/modules/cassandra-driver/cassandra-driver.test.js

```diff
@@ -41,9 +41,14 @@ const testFixtures = [
     },
     versionRanges: {
       // cassandra-driver@4.7.0 introduced a change that requires nodejs v16.9
-      // and up but previous versions support from v8 so we want to test
-      // these previous driver versions with all node versions in TAV
-      'cassandra-driver': '>=3.0.0 <4.7.0',
+      // and up but previous versions support from v8. In the fix process Cassandra
+      // team decided to drop versions from 8 to 16.
+      // Issue: https://datastax-oss.atlassian.net/browse/NODEJS-665
+      // Pull: https://github.com/datastax/nodejs-driver/pull/415#discussion_r1331010721
+      //
+      // We cannot expect new versions of the driver to work with nodejs versions <16.9
+      // therefore we want test only if node >=16.9
```

Comment:
nit: This could be a little clearer that:
1. cassandra-driver >=4.7.0 only supports node >=16, but
2. cassandra-driver 4.7.0 and 4.7.1 had a hiccup where only node >=16.9.0 worked.
Because of these complications and because Node v14 is EOL, we are skipping testing with earlier node versions.

Reply:
changed :)